contact.html

```diff
@@ -3,10 +3,11 @@
 
 <head>
     <meta charset="UTF-8">
-    <meta name="description" content="Explore my personal portfolio of front-end development projects. Discover my work as a developer, including websites and web applications I've created.">
+    <meta name="description"
+        content="Explore my personal portfolio of front-end development projects. Discover my work as a developer, including websites and web applications I've created.">
     <meta name="author" content="Mikael Elias">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Mikael Elias - Contact</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Mikael Elias - Home</title>
```

Comment:
Aqui tem que ser `Contact` mesmo
```suggestion
    <title>Mikael Elias - Contact</title>
```

Reply:
Feito!